gno.land/pkg/sdk/vm/handler.go

```diff
@@ -54,16 +45,7 @@ func (vh vmHandler) handleMsgAddPackage(ctx sdk.Context, msg MsgAddPackage) sdk.
 
 // Handle MsgCall.
 func (vh vmHandler) handleMsgCall(ctx sdk.Context, msg MsgCall) (res sdk.Result) {
-	amount, err := std.ParseCoins("1000000ugnot") // XXX calculate
-	if err != nil {
-		return abciResult(err)
-	}
-	err = vh.vm.bank.SendCoins(ctx, msg.Caller, auth.FeeCollectorAddress(), amount)
```

Comment:
These should not be removed; rather, MsgCall.Send should be parsed above instead of 1000000ugnot. 
The purpose is to deposit the coins to the package itself.
auth.FeeCollectorAddress() is wrong, it should be the package itself.

Reply:
These charges were removed because they were additional to the gas fees specified by users in Gnokey. 


For Msg_Call, the coins sent to the realm contract is managed in vm/keeper.go.

https://github.com/gnolang/gno/blob/d54ca62749f79a6992da520e1659ba0dbbdd980b/gno.land/pkg/sdk/vm/keeper.go#L239